overlord/devicestate/handlers_install.go

```diff
@@ -1278,7 +1295,19 @@ func (m *DeviceManager) doInstallSetupStorageEncryption(t *state.Task, _ *tomb.T
 		return fmt.Errorf("reading gadget information: %v", err)
 	}
 
-	encryptInfo, err := m.encryptionSupportInfo(systemAndSeeds.Model, secboot.TPMProvisionFull, systemAndSeeds.InfosByType[snap.TypeKernel], gadgetInfo)
+	var snapdVersionByType map[snap.Type]string
+	// Find snapd versions for snapd and kernel snaps in the seed for
+	// the passphrase/PINs auth checks.
+	// FDE is only supported in UC20+ (i.e. Model grade is set).
+	if volumesAuthRequired && systemAndSeeds.Model.Grade() != asserts.ModelGradeUnset {
```

Comment:
can we always attempt to get the info?
```suggestion
	if systemAndSeeds.Model.Grade() != asserts.ModelGradeUnset {
```